packages/stripe_checkout/lib/stripe_checkout.dart

```diff
@@ -12,13 +13,17 @@ Future<CheckoutResponse> redirectToCheckout({
   String? successUrl,
   String? canceledUrl,
 }) {
-  // ignore: deprecated_member_use_from_same_package
-  return stripe.redirectToCheckout(
-    context: context,
-    sessionId: sessionId,
-    publishableKey: publishableKey,
-    stripeAccountId: stripeAccountId,
-    successUrl: successUrl,
-    canceledUrl: canceledUrl,
-  );
+  if (kIsWeb) {
```

Comment:
@jonasbark  can you double check this change. Reason is that the conditional import does not work anymore with flutter 3.16.0 🤷‍♂️

Reply:
my implementation is wrong will rectify it